src/Microsoft.Identity.Web/Resource/ScopesRequiredHttpContextExtensions.cs

```diff
@@ -60,6 +61,7 @@ public static void VerifyUserHasAnyAcceptedScope(this HttpContext context, param
                     context.Response.StatusCode = (int)HttpStatusCode.Forbidden;
                     string message = string.Format(CultureInfo.InvariantCulture, IDWebErrorMessage.MissingScopes, string.Join(",", acceptedScopes));
                     context.Response.WriteAsync(message);
```

Comment:
Should this be removed, since the exception will have the message and end the response?

Reply:
The message is sent to the client (on the client side).
The exception only affect the service side
I would leave it (that would also be a breaking change for our customers)